services/idamExpressAuthenticate.js

```diff
@@ -18,12 +18,15 @@ const idamExpressAuthenticate = (args = {}) => {
       cookies.remove(res, stateCookieName);
     }
 
+    const getState = () => {
```

Comment:
what i meant was args.state was a function just like UUID is, getState would be like this
```
const getState = args.state || UUID;

const redirectUser = () => {
      const state = getState();
...
```

Reply:
I'm not sure I follow @dtaylor7 `args.state` is not a function, so `getState` would not always be a function.

Do you mean this?

```
const getState = () => args.state || UUID;

const redirectUser = () => {
      const state = getState();
...
```

Reply:
no what i mean originally was to change args.state to a function so you had more flexibility

Reply:
Fixed